fossunited/templates/foss_base.html

```diff
@@ -0,0 +1,6 @@
+{% extends "templates/base.html" %}
+
+
+{% block content %}
+    
+{% endblock %}
```

Comment:
Where are you including the css link?

Also, define the HTML here, inheriting from base.html here will does not make sense. Write the outer HTML and define blocks for pages where you will use this. 

Read Jinja docs on template extending to get a better idea...